package.json

```diff
@@ -37,7 +37,7 @@
   },
   "devDependencies": {
     "lerna": "^8.1.2",
-    "nx": "^19.2.3"
+    "nx": "^20.2.1"
```

Comment:
In general, better to avoid upgrades (especially a major version) in a feature PR 😄 But if you tested alright and Kevin and I didn't get any issue during review either, I guess we're good

Reply:
It's for github actions, it crashes because nx doesn't find its arm64 or linux package… It's not the first time and I really don't know how to fix that. Maybe we should have a fixed version instead of carret 🤔

Reply:
Agreed. I once had a similar issue locally, and then not anymore a few days later, so that's a good probability 😬